### PR TITLE
Celery upgrade

### PR DESCRIPTION
**Pull request by gcapalbo**
wip, so far the workers run and process tasks, but there are issues with using flower and autoscale

### CODE REVIEW

**Comment by snopoke** [comment]
@gcapalbo what happened to this? We need to upgrade celery in order to upgrade django...

File: requirements/requirements.txt [hunk 1]
```diff
@@ -3,15 +3,15 @@ Cython==0.17.4
 restkit==4.2.0
 jsonobject-couchdbkit==0.6.5.5
 jsonobject==0.4.3
-celery==3.0.24
+celery==3.1.11
 decorator==3.3.2
 django==1.5.6
 django-braces==1.3.1
-django-celery==3.0.23
+django-celery==3.1.10
```
Comment (snopoke) [inline requirements/requirements.txt]: Seems like v3.1 comes with django support out the box.

File: requirements/requirements.txt [hunk 2]
```diff
@@ -3,15 +3,15 @@ Cython==0.17.4
...
 django-crispy-forms==1.4.0
 django-json-field==0.5.5
-kombu==2.5.16
-billiard==2.7.3.32
+kombu==3.0.16
```
Comment (snopoke) [inline requirements/requirements.txt]: Do we need to maintain these dependencies directly? Can't we let pip sort it out?

**Comment by gcapalbo** [comment]
@snopoke This ended up dropping off my radar after boosting memory on the machine fixed the deadlock issue we were seeing. I'll get back to this and finish it up this week.

File: requirements/requirements.txt (hunk 1 above)
Comment (gcapalbo) [inline requirements/requirements.txt]: Yeah but it seemed like we still might need it for certain features. I can double-check.

File: requirements/requirements.txt (hunk 2 above)
Comment (gcapalbo) [inline requirements/requirements.txt]: They didn't update automatically when I upgraded celery, and I remember having a hard time finding the right versions of everything to make it work together, so pinning the right version is definitely the right thing to do.

**Comment by gcapalbo** [comment]
replaced by https://github.com/dimagi/commcare-hq/pull/5696

**Pull request closed without merging**